searchcore/src/vespa/searchcore/proton/server/reconfig_params.cpp

```diff
@@ -51,7 +52,7 @@ ReconfigParams::shouldIndexManagerChange() const
 bool
 ReconfigParams::shouldAttributeManagerChange() const
 {
-    return _res.attributesChanged || _res.importedFieldsChanged || _res.visibilityDelayChanged;
+    return _res.attributesChanged || _res.importedFieldsChanged || _res.visibilityDelayChanged || _res.alloc_config_changed;
```

Comment:
Should probably be tested in searchcore/src/tests/proton/documentdb/configurer/configurer_test.cpp.

Reply:
Done.